intercom/workflow.js

```diff
@@ -0,0 +1,20 @@
+const { workflow } = require("zenaton")
+
+module.exports = workflow("IntercomExample", {
+  *handle() {
+    const intercom = this.connector("intercom", "YOUR-CONNECTOR-ID");
+
+    // List users sorted by created_at DESC
+    const params = {query: {sort: 'created_at', order: 'desc'}}
+    const users = (yield intercom.get('users', params)).data.users
+
+    // Create a user
+    const payload = {
+      user_id: '125',
+      email: 'wash2@serenity.io',
+      name: 'Hoban Washburne',
+      phone: '555671243'    
```

Comment:
I'm wondering why using these user details instead of using completely fake data?

Reply:
It's mostly because those data come from the intercom doc., so it was easy to copy and paste for me.
Also I think users will easier understand how things in doc are used in the context of using Zenaton connectors.